conf/salt/project/django/run_with_db.sh

```diff
@@ -0,0 +1,5 @@
+# Shell script to set up necessary environment variables and run a PostgreSQL command
+export PGHOST="{{ pillar['env']['DB_HOST'] }}"
+export PGUSER="{{ pillar['project_name'] }}_{{ pillar['environment'] }}"
+export PGPASSWORD="{{ pillar['secrets']['DB_PASSWORD'] }}"
+exec "$@"
```

Comment:
Should we add `export PGDATABASE="{{ pillar['project_name'] }}_{{ pillar['environment'] }}"`?

Maybe also 

    if [ -n "{{ pillar['env'].get('DB_PORT') }}" ] ; then 
        export PGPORT="{{ pillar['env']['DB_PORT'] }}"; 
    fi